src/app/drivingLicence/fieldsHelper.test.js

```diff
@@ -32,17 +32,67 @@ describe("custom validation fields test", () => {
       },
     });
 
-    expect(validator(1, 38, "firstName", "middleNames")).to.be.true;
+    expect(validator(1, 38, "firstName", "DVA")).to.be.true;
```

Comment:
You have removed a param here  should it not be 
validator(1, 38, "firstName", "middleNames", "DVA")

Reply:
Need that scenario in aswell yes good spot but this scenario covers just the first name by itself

Reply:
But here your testing firstNameMiddleNameLengthValidator so it needs 5 arguments